app/views/order.py

```diff
@@ -216,52 +227,12 @@ def select_user(items):
 def get_orders(expression=None):
     orders = []
     if expression is None:
-        expression = ((datetime.now() > Order.starttime) & (Order.stoptime > datetime.now()) | (Order.stoptime == None))
+        expression = ((datetime.now() > Order.starttime) &
+                      (Order.stoptime > datetime.now()) |
+                      (Order.stoptime == None))
     if not current_user.is_anonymous():
         orders = Order.query.filter(expression).all()
     else:
-        orders = Order.query.filter((expression & (Order.public == True))).all()
+        orders = Order.query.filter(
```

Comment:
please use logical operators instead of bitwise operators

Reply:
I'd have to look up the usage of bitwise operators inside of sqlachemy queries. They override some funky stuff there so not sure if it can be replaced.

Reply:
https://docs.sqlalchemy.org/en/13/orm/tutorial.html#common-filter-operators

This is how you should filter in sqlalchemy. Normally you should use `and_` and `or_`. 
But they [override the bitwise operators](https://stackoverflow.com/a/14185275) allowing you to use those.

You can't use the normal logical `and` and `or` operators.

Reply:
So we can choose between their `and_` or `&`. 

I think the current way is okay, what do you think.

Reply:
Yes, the current way is the cleanest way to do it IMO. I looked it up, and now get why it's implemented that way: you can override the `__and__` method that does `&`, but you can't override what happens when the `and` method gets called because it converts both arguments to booleans first (https://rszalski.github.io/magicmethods/)

Reply:
Ah, I didn't look into it that far. Good to know.